package.json

```diff
@@ -16,20 +33,23 @@
     "deploy": "f () { npm run build:travis && npm run build:local && lerna exec --scope @deriv/core -- npm run deploy $@ ;}; f",
     "deploy:clean": "f () { npm run build:travis && npm run build:local && lerna exec --scope @deriv/core -- npm run deploy:clean $@ ;}; f",
     "deploy:folder": "f () { npm run build:travis && npm run build:local $1 && lerna exec --scope @deriv/core -- npm run deploy:folder $@ ;}; f",
-    "prettify": "f () { lerna exec --scope=@deriv/${1:-'*'} -- npm run --silent prettify ;}; f",
+    "prettify": "prettier --config ./.prettierrc --write \"./packages/*/{stories,script,src,build}/**/*.{js,jsx,scss}\"",
+    "precommit": "lint-staged --allow-empty",
     "publish:p2p": "f () { lerna publish from-package ;}; f",
     "serve": "f () { lerna exec --scope=@deriv/${1:-'*'} -- npm run serve $npm_config_open;}; f",
     "start": "f () { lerna exec --scope=@deriv/${1:-'*'} -- npm run start ;}; f",
-    "test": "f () { bash ./scripts/circleci-config.test.sh && lerna exec --scope=@deriv/${1:-'*'} --parallel -- npm run test ;}; f",
+    "test": "f () { bash ./scripts/circleci-config.test.sh && npm run test:stylelint  && eslint \"./packages/*/src/**/*.?(js|jsx)\" && lerna exec --scope=@deriv/${1:-'*'} --parallel -- npm run test ;}; f",
     "test:eslint": "f () { lerna exec --scope=@deriv/${1:-'*'} -- npm run test:eslint ;}; f",
```

Comment:
doesn't this one have to change as well?

Reply:
Nice catch. Indeed. I'll refactor it right away.